pkg/prowgen/prowgen.go

```diff
@@ -251,11 +257,17 @@ func generatePresubmitForTest(jobBaseBuilder *prowJobBaseBuilder, name string, i
 		},
 		Optional: opts.optional,
 	}
+	if opts.alwaysRun != nil {
+		pj.AlwaysRun = *opts.alwaysRun
+	} else {
+		pj.AlwaysRun = opts.runIfChanged == "" && opts.skipIfOnlyChanged == "" && !opts.defaultDisable && opts.pipelineRunIfChanged == ""
```

Comment:
This is hard to read. Let's put it on its function.

Reply:
What if we just use the option, otherwise false?